contracts/strategy/IStrategy.sol

```diff
@@ -58,23 +58,24 @@ interface IStrategy {
     }
 
     /**
-     * The vault linked to this stragegy
+     * Provides information about wether the strategy is synchronous or asynchronous.
+     * 
+     * @notice Synchronous strategies support instant withdrawals and deposits, 
```

Comment:
For the deposits, we don't really change the behaviour

Reply:
[fixed](https://github.com/lindy-labs/sc_solidity-contracts/pull/136/commits/47be529313b023a0fe04a06247a0095ceba141c9)